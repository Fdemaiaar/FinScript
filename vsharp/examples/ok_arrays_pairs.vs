var flows = [ -100 , 40 , 40 , 40 ] ;
var disc  = 0.10 ;
print("NPV =", npv(disc , flows)) ;
