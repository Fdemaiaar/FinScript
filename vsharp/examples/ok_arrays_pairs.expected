NPV = -0.525920
